concourse/create-build-configs/script.js

```diff
@@ -44,4 +63,7 @@ fs.readFile(versionFile, 'utf8', function(err, commit) {
 
   console.log(`Generating release id file with: ${releaseId}`);
   fs.writeFile(releaseFile, releaseId, handleErr);
+
+  console.log(`Generating books file...`);
+  fs.writeFile(booksConfigFile, JSON.stringify(books), handleErr);
```

Comment:
You're stringifying `books` twice, can you just do it once?

Reply:
it was actually worse than that, i wasn't parsing the json i was reading to begin with, so i was ending up with double encoded books